agent/src/main/scala/za/co/absa/atum/agent/model/Measurement.scala

```diff
@@ -37,8 +37,8 @@ object Measurement {
   object MeasurementProvided {
 
     private def handleSpecificType[T](
-                               measure: Measure, resultValue: T, requiredType: ResultValueType.ResultValueType
-                             ): MeasurementProvided[T] = {
+                                       measure: AtumMeasure, resultValue: T, requiredType: ResultValueType.ResultValueType
+                                     ): MeasurementProvided[T] = {
 
```

Comment:
To keep the formatting consistent:
```scala
private def handleSpecificType[T](
  measure: AtumMeasure, 
  resultValue: T, 
  requiredType: ResultValueType.ResultValueType
): MeasurementProvided[T] = {
```
Generally it should be either all in one line or like this

Reply:
I agree with Adam

Reply:
Code has been reformatted.